elixir-companies.toml

```diff
@@ -0,0 +1,1362 @@
+[[company]]
```

Comment:
Should we add the list of sections at the top?

We could use this for a test that runs in CI and check all the companies against these pre-defined sections.

Reply:
It will also make it easier for people to know which section to use for the company they're adding.

Reply:
I was _just_ thinking about this @stevedomin (sorry for the delay, was away on travel).  Perhaps we could rely on the PR template to guide them in submitting a company?